application/common/helpers/MySqlDateTime.php

```diff
@@ -3,15 +3,21 @@
 
 class MySqlDateTime
 {
-    const MYSQL_FORMAT = 'Y-m-d H:i:s';
+    const MYSQL_DATETIME_FORMAT = 'Y-m-d H:i:s';
+    const MYSQL_DATE_FORMAT     = 'Y-m-d';
```

Comment:
very possible, changes in here:  https://github.com/silinternational/idp-id-broker/commit/0a55655953b0098a7f5ec423c5835e2461e10f41